upp/configs/ghost_labels.yaml

```diff
@@ -0,0 +1,86 @@
+variables: !include variables.yaml
+global_cuts: !include simple-split.yaml
+
+
+
+ttbar: &ttbar
+  name: ttbar
+  equal_jets: False
+  pattern:
+    - "user.wlai.601589.e8547_s3797_r13144_p6215.tdd.EMPFlow_GA.25_2_14.24-06-26_med_stats_output.h5/*.h5"
+    - "user.wlai.601589.e8549_s4159_r14799_p6215.tdd.EMPFlow_GA.25_2_14.24-06-26_med_stats_output.h5/*.h5"
+    - "user.wlai.601589.e8549_s4162_r14622_p6215.tdd.EMPFlow_GA.25_2_14.24-06-26_med_stats_output.h5/*.h5"
+    - "user.wlai.601229.e8514_s4162_r14622_p6215.tdd.EMPFlow_GA.25_2_14.24-06-26_med_stats_output.h5/*.h5"
+    - "user.wlai.601230.e8514_s4162_r14622_p6215.tdd.EMPFlow_GA.25_2_14.24-06-26_med_stats_output.h5/*.h5"
+    - "user.wlai.410470.e6337_s3681_r13144_p6215.tdd.EMPFlow_GA.25_2_14.24-06-26_med_stats_output.h5/*.h5"
```

Comment:
For consistency with GN2 and to avoid excessive h5 download, I think better to avoid training on the [new-ttbar-run3](https://ftag.docs.cern.ch/samples/samples/) samples, and stick to the equivalent samples to https://github.com/umami-hep/umami-preprocessing/blob/main/upp/configs/single-b.yaml#L11, what do you think?

Reply:
Sounds good